hack/demo-crds.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright The Conforma Contributors
```

Comment:
You can pull these in from the source using kustomize. That way we always use the latest when testing.

https://github.com/conforma/knative-service/pull/27/files#diff-f8244994e5b1e8731d8b1ab0f7cb1852027dba27097324d18076e88f16b8e088R26-R28